### PR TITLE
zh-CN: create `management.getPermissionWarningsByManifest()`

### PR DESCRIPTION
**Pull request by PassionPenguin**

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/management/getpermissionwarningsbymanifest/index.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+slug: Mozilla/Add-ons/WebExtensions/API/management/getPermissionWarningsByManifest
+l10n:
+  sourceCommit: b8a0743ca8b1e1b1b1a95cc93a4413c020f11262
+---
+
+{{AddonSidebar}}
+
+当用户安装或升级附加组件时，浏览器可能会警告用户有关附加组件请求的任何特别强大的[权限](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/permissions)。并非所有权限都会导致警告，而且此行为在各个浏览器中并不标准化。
+
+给定 [manifest.json](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json) 文件的文本，此函数以字符串数组形式返回生成的附加组件的权限警告。
```
Comment (JasonLamv-t) [inline files/zh-cn/mozilla/add-ons/webextensions/api/management/getpermissionwarningsbymanifest/index.md:12]:
```suggestion
给定 [manifest.json](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json) 文件的文本，此函数以字符串数组形式返回针对附加组件的权限警告。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/management/getpermissionwarningsbymanifest/index.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+```js-nolint
+let gettingWarnings = browser.management.getPermissionWarningsByManifest(
+  manifestString      // 字符串
+)
+```
+
+### 参数
+
+- `manifestString`
+  - : `string`，包含清单文件的字符串。这必须是有效的清单：比如说它必须包含所有强制性的清单键。
```
Comment (JasonLamv-t) [inline files/zh-cn/mozilla/add-ons/webextensions/api/management/getpermissionwarningsbymanifest/index.md:29]:
```suggestion
  - : `string`，包含清单文件的字符串。必须是有效的清单：比如说它必须包含所有必须的清单键。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/management/getpermissionwarningsbymanifest/index.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+[`Promise`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise)，将用一个字符串数组兑现，其中各字符串都包含权限警告的文本。
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 示例
+
+记录给定清单文件的权限警告：
```
Comment (JasonLamv-t) [inline files/zh-cn/mozilla/add-ons/webextensions/api/management/getpermissionwarningsbymanifest/index.md:41]:
```suggestion
打印给定清单文件的权限警告：
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Well done! Need some improvements.

File: files/zh-cn/mozilla/add-ons/webextensions/api/management/getpermissionwarningsbymanifest/index.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+```js-nolint
+let gettingWarnings = browser.management.getPermissionWarningsByManifest(
+  manifestString      // 字符串
+)
+```
+
+### 参数
+
+- `manifestString`
+  - : `string`，包含清单文件的字符串。这必须是有效的清单，例如说它必须包含所有强制性的清单键。
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/management/getpermissionwarningsbymanifest/index.md:29]:
```suggestion
  - : `string`，包含清单文件的字符串。必须是有效的清单，例如它必须包含所有强制性的清单键。
```

**Review by JasonLamv-t — APPROVED** [review]
LGTM.

**Pull request merged**